### PR TITLE
Adding posted_time to job search response

### PR DESCRIPTION
**Pull request by David-Sharpe**

### CODE REVIEW

File: lib/cb/models/implementations/job.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Job < ApiResponseModel
                     :location, :distance, :latitude, :longitude, :location_formatted, :location_metro_city,
                     :description, :requirements, :employment_type,
                     :details_url, :service_url, :similar_jobs_url, :apply_url,
-                    :begin_date, :end_date, :posted_date,
+                    :begin_date, :end_date, :posted_date, :posted_time,
```
Comment (nivardus) [inline lib/cb/models/implementations/job.rb]: Could we just combine `posted_date` and `posted_time` into a DateTime?

Comment (bowersaa) [inline lib/cb/models/implementations/job.rb]: Would that cause problems in differences in anticipated data format down the line?

Comment (nivardus) [inline lib/cb/models/implementations/job.rb]: Undoubtably!

Comment (ausmith) [inline lib/cb/models/implementations/job.rb]: I don't know what our limitations are to know if that'd hurt us with data format. I'd prefer it be a DateTime as well if we can.

Comment (ausmith) [inline lib/cb/models/implementations/job.rb]: We going to test switching to a DateTime prior to merging this guy or just move forward with this?

**Comment by nivardus** [comment]
Having date and time listed separately is a odd. However, as this is the format returned by the API it may be better to maintain the same properties all the way down.

**Comment by ausmith** [comment]
@nivardus :+1: fair enough

**Pull request merged**